scripts/examples/gn_efr32_example.sh

```diff
@@ -98,6 +98,12 @@ if [ "$#" == "0" ]; then
             Periodic query timeout variable for OTA in seconds
         rs91x_wpa3_only
             Support for WPA3 only mode on RS91x
+        sl_matter_version
+            Overwrite Matter software version
+        sl_matter_version_str
+            Overwrite Matter software version string
```

Comment:
Does this option require passing a version value? If yes then perhaps a different wording would be helpful. If not then how does the user supply the value? Same for the hardware version